Bottom: 80beb43c2145f159ed0d8f2da705f88441384d87
Top:    f340598fe5924b4e4c5011d98a4788eaf853d104
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2013-06-05 12:07:57 +0100

Add warning about bogus code in strflags.c


---

diff --git a/src/strflags.c b/src/strflags.c
index 3e57696..d6d6273 100644
--- a/src/strflags.c
+++ b/src/strflags.c
@@ -576,6 +576,7 @@ dump_flag (FlagType * f)
 {
   int l;
   printf ("F:%08x T:[", f->f);
+#warning THIS NEXT LINE IS BOGUS, THE ARRAY IS HALF THIS LENGTH
   for (l = 0; l < (MAX_LAYER + 7) / 8; l++)
     printf (" %02x", f->t[l]);
   printf ("]");
